lib/active_admin/views/components/columns.rb

```diff
@@ -93,15 +93,21 @@ def calculate_columns!
         all_margins_width = margin_size * (span_count - 1)
         column_width = (100.00 - all_margins_width) / span_count
 
-        children.each_with_index do |col, i|
-          is_last_column = i == (columns_count - 1)
-          col.set_column_styles(column_width, margin_size, is_last_column)
+        children.each_with_index do |child, i|
+          if child.is_a?(Column)
+            is_last_column = i == (columns_count - 1)
+            child.set_column_styles(column_width, margin_size, is_last_column)
+          end
         end
       end
 
       def columns_span_count
         count = 0
-        children.each {|column| count += column.span_size }
+        children.each do|child|
+          if child.is_a?(Column)
+            count += child.span_size
+          end
+        end
```

Comment:
You can simplify Line 97 and Line 107 by building a `columns` method, wich looks like:
```ruby
def columns
  children.select { |child| child.is_a?(Column) } 
end
```